processor/src/test/kotlin/com/syouth/kmapper/processor/convertors/DataClassTypeConverterTest.kt

```diff
@@ -67,6 +59,29 @@ internal class DataClassTypeConverterTest {
         Assertions.assertTrue(converter.isSupported(fromDataType, toDataType.makeNullable(), PathHolder()))
     }
 
+    @Test
```

Comment:
We also need some more tests that test specifically that POJO<->POJO and Data<->Pojo works correctly.

Reply:
I fully agree.
However, do you mind if I delegated this to you? These tests are probably fare easier constructed with a more internal knowledge of the conversion.

Reply:
I will handle that

Reply:
@cyc1ingsir Please allow me to modify your branch, I can't push my changes.

Reply:
This is allowed :thinking: 
![image](https://user-images.githubusercontent.com/2297037/222929180-349eac17-c192-4953-866c-97ddf5ca0e09.png)
However, I've added you as collaborator to my forked repository. You should have received an invite. Hopefully, this should work.